packages/code-du-travail-frontend/src/contributions/rehypeToReact.js

```diff
@@ -24,18 +24,19 @@ const Hdn = (props) => (
   </Alert>
 );
 
-const ContentSP = ({ raw }) => (
-  <>
-    {raw && (
-      <StyledContent>
-        <FicheServicePublic data={JSON.parse(raw).children} />
-      </StyledContent>
-    )}
-  </>
-);
+const ContentSP = ({ raw }) => {
+  return (
```

Comment:
Why did you add a return here? There is an issue with this syntax?

Reply:
I have added a console log :) and I prefer this syntaxe since it's easier to add log statement or code before